source/_integrations/kulersky.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: Kuler Sky
+description: Instructions for integrating Brightech Kuler Sky Bluetooth floor lamps with Home Assistant.
+ha_category:
+  - Light
+ha_iot_class: Local Polling
+ha_release: '0.118'
```

Comment:
```suggestion
ha_release: '1.0.0'
```

Reply:
This should have been 1.0. We don't specify patch versions in the releases...